.github/workflows/pytest-workflow.yml

```diff
@@ -107,17 +107,17 @@ jobs:
           - profile: "conda"
             tags: scimap/mcmicro
           - profile: "conda"
+            tags: universc
+          - profile: "singularity"
+            tags: universc
+          - profile: "singularity"
             tags: sentieon/bwaindex
-          - profile: "conda"
+          - profile: "singularity"
             tags: sentieon/bwamem
-          - profile: "conda"
+          - profile: "singularity"
```

Comment:
What is happening there?

Reply:
I can't run Sentieon with the singularity images introduced in this PR. I get the now well-known:
```
error while loading shared libraries: libstdc++.so.6: cannot open shared object file: No such file or directory
```
Similar issue with the subsequent images made by Don for Sentieon 202112.07.

I'm afraid this is a showstopper for this PR 😬